onnx/defs/controlflow/defs.cc

```diff
@@ -472,13 +472,13 @@ C-style code:
 
     graph body-net (
       %i[INT32, scalar]
-      %keepgoing[BOOL, scalar]
-      %b[INT32, scalar]
+      %keepgoing_in[BOOL, scalar]
+      %b_in[INT32, scalar]
     ) {
       %my_local = Add(%a, %b)
-      %b_out = Sub(%a, %b)
+      %b_out = Sub(%a, %b_in)
       %keepgoing_out = Greater(%my_local, %b_out)
-      %user_defined_vals = Add(%b, %b)
+      %user_defined_vals = Add(%b_in, %b_in)
```

Comment:
I think this should be called user_defined_val. It's in the Loop implementation not the loop body that these get aggregated (one item per loop iteration) and returned as the user_defined_vals output.